src/Destructurama.Attributed/Destructurama.Attributed.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
You can avoid multitargeting and referencing system.componentmodel.annotations in the same way as it was done for Microsoft.Extensions.Logging.LogPropertyIgnoreAttribute.

Reply:
OK, I'll have a look at it.

Reply:
Multitargeting removed. However, some testing did show, that with this implementation attributes requiring access to property values will not work. This would require to implement something that provides GetValue from data class instead of meta class. I suggest to wait until an issue is submitted as this may require much more code changes than we have now.

Reply:
> attributes requiring access to property values will not work

?